mysite/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for mysite project.
+
+Generated by 'django-admin startproject' using Django 3.2.18.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+load_dotenv(os.path.join(BASE_DIR, '.env'))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
```

Comment:
Looks like this this happened too early too.. I haven't yet selected where we'd be deploying this. I'll likely choose a PaaS that's close to what Heroku used to provide before it stopped offering free service.

Reply:
Resolving as this is solved.